org.scala-ide.sdt.core/src/org/scalaide/core/compiler/DeclarationPrinter.scala

```diff
@@ -176,7 +177,9 @@ abstract class DeclarationPrinter extends HasLogger {
     if (tpe.termSymbol.hasPackageFlag) ""
     else {
       val prefix = showType(tpe.pre)
-      if (prefix != "") prefix + "." + tpe.sym.nameString
+      import rootMirror.getPackageIfDefined
+      val pkg = getPackageIfDefined(TermName(prefix))
+      if (prefix != "" && pkg != definitions.ScalaPackage) prefix + "." + tpe.sym.nameString
```

Comment:
I'm not sure if I like like this change. When is it useful to not display the `scala` package?